api/data_refinery_api/views.py

```diff
@@ -152,21 +152,19 @@ class SearchAndFilter(generics.ListAPIView):
 
     """
 
-    # Only Experiments with processed objects are exposed
-    queryset = Experiment.processed_public_objects.annotate(samples_count=Count('samples')).all()
-
-    # For developing, you can uncomment this to expose everything.
-    #queryset = Experiment.objects.annotate(samples_count=Count('samples')).all()
-
     serializer_class = ExperimentSerializer
     pagination_class = LimitOffsetPagination
 
     filter_backends = (NoMarkupDjangoFilterBackend, filters.SearchFilter, filters.OrderingFilter)
     filter_class = ExperimentFilter
 
+    # Ordering
+    ordering_fields = ('total_samples_count', 'id', 'created_at',
+                       'source_first_published', 'accession_code',)
+    ordering = ('-total_samples_count',)
+
```

Comment:
The ordering section is now repeated, so this code doesn't do anything

Reply:
good catch, not sure why this happened. I'll remove it